enginetest/queries/queries.go

```diff
@@ -742,6 +742,23 @@ var SpatialQueryTests = []QueryTest{
 }
 
 var QueryTests = []QueryTest{
+	{
+		Query: "select * from (select 1, 1) a(x,y) union select * from (select 1, NULL) b(x,y) union select * from (select 1,1) c(x,y);",
```

Comment:
Seems like the patch below is only going to work with literal NULL values (nothing else is types.Null)

Need a test where you're doing the union selects from table values that have null in them

Reply:
i think that's unrelated? a nullable column won't have sql.NullType